src/cmr/concepts/collection.js

```diff
@@ -17,37 +17,41 @@ export default class Collection extends Concept {
   }
 
   /**
-   * Set a value in the result set that a query has not requested but is neccessary for other functionality
+   * Set a value in the result set that a query has not requested but is necessary for other functionality
    * @param {String} id Concept ID to set a value for within the result set
    * @param {Object} item The item returned from the CMR json endpoint
    */
   setEssentialJsonValues(id, item) {
     super.setEssentialJsonValues(id, item)
 
-    // Associations are used by services and variables, its required to correctly
-    // retrieve those objects and shouldn't need to be provided by the client
-    const { associations } = item
+    const { association_details: associationDetails } = item
+
+    const formattedAssociationDetails = camelcaseKeys(associationDetails, { deep: true })
 
-    if (associations) {
-      this.setItemValue(id, 'associations', associations)
+    // Associations are used by services, tools, and variables, its required to correctly
```

Comment:
```suggestion
    // Associations are used by services, tools, and variables, it's required to correctly
```
its = possessive. it's = it is